requirements.txt

```diff
@@ -6,6 +6,7 @@ django-cors-middleware==1.3.1
 django-test-without-migrations==0.4
 djangorestframework==3.5.3
 gunicorn==19.6.0
+pandas==0.19.2
```

Comment:
The problem is that I really have no idea what was happening. I'm using anaconda to run it now. Do you think we should add some note on README explaining that panda's issues can be solved with anaconda?